connect/mirror/README.md

```diff
@@ -141,7 +141,38 @@ nearby clusters.
 N.B. that the `--clusters` parameter is not technically required here. MM2 will work fine without it; however, throughput may suffer from "producer lag" between
 data centers, and you may incur unnecessary data transfer costs.
 
-## Shared configuration
+## Configuration
+### General Kafka Connect Config
+All Kafka Connect, Source Connector, Sink Connector configs, as defined in [Kafka official doc] (https://kafka.apache.org/documentation/#connectconfigs), can be 
+directly used in MM2 configuration without prefix in the configuration name. As the starting point, most of these configs may work well with the exception of `tasks.max`.
```

Comment:
I think you are referring to the default values of the Connect configuration properties -- maybe use "default" in this sentence to make that more clear.

Reply:
done